app/src/main/java/com/clearkeep/screen/chat/group_create/CreateGroupActivity.kt

```diff
@@ -49,7 +51,9 @@ class CreateGroupActivity : AppCompatActivity() {
                     composable("invite_group") {
                         inviteGroupViewModel.updateContactList()
                         InviteGroupScreen(
+                            InviteMember,
                             inviteGroupViewModel,
+                            listMemberInGroup= listOf<User>(),
```

Comment:
remove this. and add listOf<User>() to InviteGroupScreen

Reply:
fixed